base/loading.jl

```diff
@@ -492,6 +535,16 @@ function create_expr_cache(input::AbstractString, output::AbstractString)
 end
 
 compilecache(mod::Symbol) = compilecache(string(mod))
+
+"""
+    Base.compilecache(module::String)
+
+Creates a precompiled cache file for module (see help for [`require`](:func:`require`)) and all of its
```

Comment:
`compilecache` docstring: is this cross-reference wrong? the help for `require` doesn't seem to mention what precompiled cache files are